### ISSUE TITLE
API无法逐字输出

### ISSUE THREAD

**Issue by hsiao417** [THUDM/ChatGLM-6B#1099]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

请我我用.net 有两个问题点
1、无法逐字读出
2、无法记录历史对话记录，请问这两个可以决解吗

以下是我当前写的代码

 Public Async Sub CallAPI()
        Dim promptStr As String = TextBox1.Text
        Dim url As String = "http://175.178.xxx.xxx:8000"
        Dim client As New HttpClient()

        Dim jsonObject As New With {
            .prompt = promptStr,
            .hsitory1 = ""' history
        }

        Dim content = New StringContent(JsonConvert.SerializeObject(jsonObject), Encoding.UTF8, "application/json")

        Dim response As HttpResponseMessage = Await client.PostAsync(url, content)

        If response.IsSuccessStatusCode Then
            Dim responseBody As String = Await response.Content.ReadAsStringAsync()

            Console.WriteLine(responseBody)
            TextBox2.Text = responseBody
            ' history.Add(responseBody)
        Else
            Console.WriteLine("HTTP Status: " + response.StatusCode.ToString)
        End If

    End Sub

### Steps To Reproduce

同上

### Environment

```markdown
- OS:Ubuntu 19
- Python:
- Transformers:
- PyTorch:
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
```
